k8s/clusterrole.go

```diff
@@ -0,0 +1,37 @@
+package k8s
+
+import (
+	"bytes"
+
+	rbacv1 "k8s.io/api/rbac/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	yamlutil "k8s.io/apimachinery/pkg/util/yaml"
+	"k8s.io/client-go/kubernetes"
+)
+
+func ApplySystemNodeClusterRoleBinding(kubeConfigPath string) error {
```

Comment:
We shouldn't have this in k8s package, k8s package should only have wrapper functions to create functions, this should be in a separate package like `authorization` to include the apply and static manifests as constants